app/controllers/books_controller.rb

```diff
@@ -5,7 +5,7 @@ class BooksController < ApplicationController
 
   # GET /books or /books.json
   def index
-    @books = Book.all
+    @books = Book.order('id ASC').page(params[:page])
```

Comment:
```suggestion
    @books = Book.order(:id).page(params[:page])
```

ASCはデフォルトなので省略することが多いですね